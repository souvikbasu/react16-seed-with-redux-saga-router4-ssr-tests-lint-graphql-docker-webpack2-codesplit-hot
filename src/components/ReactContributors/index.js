import React from 'react'
import { Link } from 'react-router-dom'

const ReactContributors = ({contributors, onFetchContributors}) => (
<div>
    <button onClick={onFetchContributors}>Get React Contributors</button>
    <div>
    {contributors ? (
    contributors.map(c => (
        <div key={c.id}>
        <Link to={`/contributors/${c.login}`}>
            {c.login}
        </Link>
        </div>
    ))
    ) : (
    <div>Loading...</div>
    )
    }
    </div>
</div>
);

export default ReactContributors;