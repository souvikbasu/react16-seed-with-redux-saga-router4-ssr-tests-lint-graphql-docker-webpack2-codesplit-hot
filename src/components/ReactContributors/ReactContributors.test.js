import React from "react";
import { shallow } from "enzyme";

import ReactContributors from "./";

describe("<ReactContributors />", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<ReactContributors />);
    expect(wrapper).toMatchSnapshot();
  });
});
