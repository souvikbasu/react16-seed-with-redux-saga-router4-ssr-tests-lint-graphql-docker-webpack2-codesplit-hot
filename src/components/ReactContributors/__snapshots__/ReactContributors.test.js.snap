// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReactContributors /> matches snapshot 1`] = `
<div>
  <button>
    Get React Contributors
  </button>
  <div>
    <div>
      Loading...
    </div>
  </div>
</div>
`;
